docs/community_features.md

```diff
@@ -75,6 +75,12 @@ Here is a list of features that have been added to the firmware as a list, group
  - ([#221]) The normal behavior of the Deluge is to try to keep up with 'in progress' notes when instant switching between clips by playing them late. However this leads to glitches with drum clips and other percussive sounds. Changing this setting to OFF will prevent this behavior and *not* try to keep up with those notes, leading to smoother instant switching between clips.
 	- This feature can be turned ON/OFF in the Runtime Settings (Community Features) Menu (accessed by pressing "SHIFT" + "SELECT"). 
 
+#### 4.1.5 - Fill Clips
```

Comment:
Just a small suggested change.

Can you move your addition up so that it becomes #### 4.1.3 - Fill Clips and then renumber the other two so they become #### 4.1.4 - New Grid Layout and #### 4.1.5 - Catch Notes